demisto_sdk/commands/common/constants.py

```diff
@@ -932,7 +933,7 @@ class PB_Status:
     'classifier': JSON_ALL_CLASSIFIER_REGEXES,
     'layout': JSON_ALL_LAYOUT_REGEXES,
     'incidentfield': JSON_ALL_INCIDENT_FIELD_REGEXES + JSON_ALL_INDICATOR_FIELDS_REGEXES,
-    'incidenttype': [INCIDENT_TYPE_REGEX],
+    'incidenttype': JSON_ALL_INCIDENT_TYPES_REGEXES,
     'image': [IMAGE_REGEX],
     'reputation': [REPUTATION_REGEX],
```

Comment:
the reputations have moved to here: `https://github.com/demisto/content/tree/master/Packs/Base/Misc`
please validate that we are using the correct regexes.

Reply:
Changed it in constants of MiscDir